mbedtls/src/error.rs

```diff
@@ -23,66 +24,114 @@ pub trait IntoResult: Sized {
     }
 }
 
+pub mod codes {
+    pub use super::HiError::*;
+    pub use super::LoError::*;
+}
+
 // This is intended not to overlap with mbedtls error codes. Utf8Error is
 // generated in the bindings when converting to rust UTF-8 strings. Only in rare
 // circumstances (callbacks from mbedtls to rust) do we need to pass a Utf8Error
 // back in to mbedtls.
 pub const ERR_UTF8_INVALID: c_int = -0x10000;
 
 macro_rules! error_enum {
-    {enum $n:ident {$($rust:ident = $c:ident,)*}} => {
-        #[derive(Debug, Eq, PartialEq)]
+    {
+        const MASK: c_int = $mask:literal;
+        enum $n:ident {$($rust:ident = $c:ident,)*}
```

Comment:
nit: `$n` could be called `$error` or something else. 

(This is also part of the original PR, we can just ignore this but I just wanted to object about it)

Reply:
addressed this. `$n -> $error`